pkg/services/kmsproviders/osskmsproviders/osskmsproviders.go

```diff
@@ -24,10 +24,6 @@ func ProvideService(enc encryption.Internal, settings setting.Provider, features
 }
 
 func (s Service) Provide() (map[secrets.ProviderID]secrets.Provider, error) {
-	if s.features.IsEnabled(featuremgmt.FlagDisableEnvelopeEncryption) {
```

Comment:
We delegate that behavior to the `Provide` caller in order to unify all the usages of `featuremgmt.FlagDisableEnvelopeEncryption`.